modules/ROOT/pages/proc_installing-pipelines-operator-via-yaml.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * assembly_installing_pipelines.adoc
+
+[id="proc_installing-pipelines-operator-via-yaml.adoc_{context}"]
+= Installing OpenShift Pipelines from the OperatorHub using the CLI
+
+You can install OpenShift Pipelines from the OperatorHub using the CLI.
+
+.Prerequisites
+
+- Access to an OpenShift cluster using an admin account.
+
+- *oc* is installed on your system.
+
+.Procedure
+
+. Create an OperatorGroup object YAML file - `operatorgroup.yaml`:
++
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-pipelines-operator	
+  namespace: openshift-marketplace
+spec:
+  targetNamespaces:
+  - openshift-marketplace
+----
```

Comment:
we don't need this step. As our operator is installed in AllNamespace install mode, the operatorgroup will be all ready present in `openshift-operators` namespace.

When installing from ui the namespace is automatically set to openshift-operators (cannot change for AllNamespace install mode)

Reply:
Removed!